src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1187,11 +1187,27 @@ Prevent Long URL From Breaking Out of Container
 /*
 Glyphicon for success status messages
```

Comment:
Can follow the big block heading style used above (refer to the `Responsiveness` section).